recipes/lora_finetune_single_device.py

```diff
@@ -282,10 +284,12 @@ def setup(self, cfg: DictConfig) -> None:
 
         # Dataloader depends on the tokenizer and loss_fn and should be
         # setup after all of these are setup
+        collate_name = cfg.get("collate_fn", "torchtune.data.padded_collate_sft")
```

Comment:
Why is padded_collate_sft the default? I thought that didn't work

Reply:
That's the standard collate for finetuning text models.